package.json

```diff
@@ -12,7 +12,7 @@
     "npm": ">=8.0.0"
   },
   "config": {
-    "DOCKER_SERVER": ""
+    "DOCKER_SERVER": "harbor.library.tamu.edu/weaver/wvr-components"
```

Comment:
I don't think we are supposed to have a default for this.
I imagine that you accidentally committed this.
Please remove.

Reply:
This should now be resolved @kaladay